set-versions

```diff
@@ -38,7 +38,7 @@ for project in $(find . -name project.clj -exec dirname '{}' \;); do
 
   cljproj=$(basename $project)
 
-  if [ "$cljproj" == "Donkey" ]; then
+  if [ "$cljproj" == "Terrain" ]; then
```

Comment:
continuing my bad habit of out-of-scope comments, I don't think we use this file any more and might want to remove it. not sure who would be the person to confirm/deny this